packages/bpk-svgs/gulpfile.babel.js

```diff
@@ -317,27 +318,22 @@ gulp.task('icons-font', ['icons-common'], () => {
 
   const saveMapping = generateFont.on('glyphs', glyphs => {
     const baseDir = 'dist/font';
-    // Og all the glyphs generate a key value pair with name and code
     const mapping = glyphs.reduce((acc, glyph) => {
-      // use punycode to get the text representation of the unicode
```

Comment:
I don't agree on removing this comment, looking at the code below it's not clear what's happening, at least the comment explains the output

Reply:
I've added it back in